src/app/analise/Analise.tsx

```
@@ -1,87 +1,110 @@
+							antigo Previne Brasil. As informações permanecem disponíveis para
+							consulta, mas é importante ressaltar que, com o encerramento do
+							programa, os resultados apresentados não devem ser considerados
+							para o cofinanciamento da Atenção Primária à Saúde.
+						</span>
+					}
+				/>
+			</div>
+
+			<div className="cardGrid">
+				{cardsData.map((card, index) => (
+					<div key={index} className="cardContainer">
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)